docs/guide/examples.rst

```diff
@@ -470,7 +470,7 @@ The parking env is a goal-conditioned continuous control task, in which the vehi
 
 .. note::
 
-	the hyperparameters in the following example were optimized for that environment.
```

Comment:
Something happened in this file? Indents are only 2-spaces, while rest of the file uses standard 4-spaces.

Reply:
in fact (and we should change that at some point), the rest of the file uses 2-spaces indent, I did a tab to spaces conversion